operator/Makefile

```diff
@@ -1,33 +1,56 @@
 # Details about the Kiali operator image.
-OPERATOR_IMAGE_NAME ?= kiali/kiali-operator
+OPERATOR_IMAGE_NAME ?= quay.io/kiali/kiali-operator
 OPERATOR_IMAGE_VERSION ?= dev
 OPERATOR_NAMESPACE = kiali-operator
 
-# When deploying the Kiali operator, this setting will indicate if it should install Kiali also.
-OPERATOR_INSTALL_KIALI ?= true
+# When deploying the Kiali operator via operator-create target, this indicates if it should install Kiali also.
+OPERATOR_INSTALL_KIALI ?= false
 
 # When installing Kiali, here are some configuration settings for it.
 AUTH_STRATEGY ?= openshift
 CREDENTIALS_USERNAME ?= admin
 CREDENTIALS_PASSPHRASE ?= admin
 IMAGE_VERSION ?= dev
 NAMESPACE ?= istio-system
+VERBOSE_MODE ?= 3
 
-# Build the Kiali operator container image
-operator-build:
+# Find the client executable (either istiooc or oc or kubectl)
+OC ?= $(shell which istiooc 2>/dev/null || which oc 2>/dev/null || which kubectl 2>/dev/null || echo "MISSING-OC/KUBECTL-FROM-PATH")
+
+.PHONY: help
+all: help
+help: Makefile
+	@echo
+	@echo "Choose a make target to run:"
+	@echo
+	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
+	@echo
+
+.ensure-operator-sdk-exists:
+	@$(eval OP_SDK ?= $(shell which operator-sdk 2>/dev/null || echo -n ""))
+	@if [ "${OP_SDK}" == "" ]; then \
+	  echo "ERROR: You need operator-sdk in your PATH. You can get it from https://github.com/operator-framework/operator-sdk/releases"; \
+	  exit 1; \
+	fi
+
+## operator-build: Build the Kiali operator container image
+# Requires operator-sdk - download it from https://github.com/operator-framework/operator-sdk/releases
+operator-build: .ensure-operator-sdk-exists
 	@echo Build operator
-	operator-sdk build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
+	${OP_SDK} build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
 
-# Push the Kiali operator container image to a remote repo
+## operator-push: Push the Kiali operator container image to a remote repo
 operator-push:
 	@echo Push operator image to image repo
 	docker push ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
 
-# Deploy the Kiali operator to the cluster using the install script.
-# The Kiali Operator does not create secrets, but this calls the install
-# script which creates a Kiali secret for you as a convienence
-# so you don't have to remember to do it yourself.
-operator-deploy: operator-remove
+## operator-create: Deploy the Kiali operator to the cluster using the install script.
+# By default, this target will not deploy Kiali - it will only deploy the operator.
+# You can tell it to also install Kiali by setting OPERATOR_INSTALL_KIALII=true.
+# The Kiali operator does not create secrets, but this calls the install script
+# which can create a Kiali secret for you as a convienence so you don't have
+# to remember to do it yourself. It will only do this if it was told to install Kiali.
+operator-create: operator-delete
```

Comment:
@jmazzitelli This can cause "unable to create new content in namespace kiali-operator because it is being terminated". _operator-delete_ deletes namespace but it does not block so _deploy-kiali-operator.sh_ is trying to use the namespace which can still being terminated (it's not happening always).

Reply:
Ok @FilipB that's a good reason for me to remove this. I only added this delete dependent target as a convenience. Maybe I'll have the create target fail if it already exists. Thanks.

Reply:
This is fixed - I changed it so the operator-create target will not execute until the operator namespace has finished being deleted.